config.go

```diff
@@ -49,6 +48,7 @@ var (
 	btcwalletHomeDir   = btcutil.AppDataDir("btcwallet", false)
 	btcdHomedirCAFile  = filepath.Join(btcdHomeDir, "rpc.cert")
 	defaultConfigFile  = filepath.Join(btcwalletHomeDir, defaultConfigFilename)
+	defaultPubPassphrase = "public"
```

Comment:
Could you please elaborate? Do you want defaultPubPassphrase only to be exported from another package? 

go fmt - addressed.